aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -56,6 +57,10 @@ extern "C" void dgesdd_(char *jobz, int *m, int *n, double *a, int *lda,
                         double *s, double *u, int *ldu, double *vt, int *ldvt, double *work, int *lwork, int *iwork, int *info);
 extern "C" void sgesdd_(char *jobz, int *m, int *n, float *a, int *lda,
                         float *s, float *u, int *ldu, float *vt, int *ldvt, float *work, int *lwork, int *iwork, int *info);
+
+// getrs
+extern "C" void dgetrs_(char *trans, int *n, int *nrhs, double *a, int *lda, int *ipiv, double *b, int *ldb, int *info);
```

Comment:
Do you know where our lapack is coming from? From reading random online sources (https://software.intel.com/en-us/mkl-developer-reference-c-getrs) I think the first arg should be a `char`, but maybe it's actually supposed to be a `char *` (and take a string literal) depending on what we're linking with

Reply:
Note that those are LAPACKE functions (with an extra `int matrix_layout`, too), openblas has the function as declared here:
https://github.com/xianyi/OpenBLAS/blob/6b41eb9c0cf1ce8c8a10a8e7fd3a6f8cbd94a14a/common_interface.h#L675
I can't find it in the online doc for MKL, but the dgetrs declaration in my installed `mkl_lapack.h` is also with `char*`.
(Reminds me of the only tangentially related https://lwn.net/Articles/791393/, but that's not of worry now.)